.triggerbuild

```diff
@@ -0,0 +1 @@
+Thu May  5 08:35:06 UTC 2022
```

Comment:
Should we use a more maintainable time format instead?

Reply:
@elgnay this value comes from `date > .triggerbuild`, do you want me to change the format like `2022-05-05`?

Reply:
yes, that looks better.